.python-version

```diff
@@ -0,0 +1 @@
+3.10.4
```

Comment:
I am indeed using pyenv, at least since we upgraded to 3.10. Given that Python 3.10 is a pinned project dependency, it makes sense to explicitly set a version here so that other people can immediately get the correct version if they're also using pyenv.

Reply:
I think because it's such a specific point version of Python - that's the gist of people's reasons why not to include it.  Not used pyenv before myself so I just wanted to see what other people do out there.

Reply:
Happy to leave it!